openandroidinstaller/assets/configs/kminilte.yaml

```diff
@@ -0,0 +1,28 @@
+metadata:
+  maintainer: Ludovic Rousseau
+  device_name: Samsung Galaxy S5 mini
+  is_ab_device: false
+  device_code: kminilte
+  supported_device_codes:
+    - kminilte
+    - kminiltexx
+steps:
```

Comment:
Ideally you should also add an `android` `requirement` ^^

Reply:
> android: [OPTIONAL] int|str; Android version to install prior to installing a custom ROM.

I have no idea what the answer should be here.
The phone was originally provided with 4.4 KitKat but I did not note down the version I had before the reflash.

Reply:
I don't know if an Android version newer than 4.4 is required.
I propose to let this field empty.

Reply:
That is also fine and it should not pose any issues at the age of the device :+1: